### PR TITLE
[DT-148][risk=no] Update menu items for nightly genomics test

### PR DESCRIPTION
**Pull request by dolbeew**
Update `genomic-extraction-to-vcf` test to handle updated WGS menu item
Screen Shot 2022-11-28 at 1 28 47 PM

### CODE REVIEW

File: e2e/app/text-labels.ts [hunk 1]
```diff
@@ -40,6 +40,7 @@ export enum MenuOption {
   Fitbit = 'Fitbit',
   GenderIdentity = 'Gender Identity',
   LabsAndMeasurements = 'Labs and Measurements',
+  LongReadWGS = 'Long Read WGS',
```
Comment (dolbeew) [inline e2e/app/text-labels.ts:43]: Added this for the new Long Read menu item. Not used currently but will most likely need it in the future.

File: e2e/app/text-labels.ts [hunk 2]
```diff
@@ -55,7 +56,7 @@ export enum MenuOption {
   SuppressCriteriaFromTotalCount = 'Suppress criteria from total count',
   Surveys = 'Surveys',
   Visits = 'Visits',
-  WholeGenomeVariant = 'Whole Genome Sequence'
+  ShortReadWGS = 'Short Read WGS'
```
Comment (jmthibault79) [inline e2e/app/text-labels.ts:59]: nit: let's keep this alphabetized

**Review by jmthibault79 — APPROVED** [review]
Thanks!

File: e2e/app/text-labels.ts (hunk 2 above)
Comment (dolbeew) [reply e2e/app/text-labels.ts]: Good catch. I moved the `LongReadWGS` up and forgot to move this one after renaming

**Pull request merged**